include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -233,7 +233,7 @@ void annotatePath(const FacadeT &facade,
                      extractor::guidance::TurnInstruction::NO_TURN(),
                      {{0, INVALID_LANEID}, INVALID_LANE_DESCRIPTIONID},
                      facade.GetTravelMode(target_node_id),
-                     INVALID_ENTRY_CLASSID,
+                     {},
```

Comment:
These defines are something @daniel-j-h always pushed me to do and I quite like them by now. Do you think that it would be possible to preserve some naming here, instead of anonymous braces. We could of course use comments as well.